lib/hsd.js

```diff
@@ -95,6 +95,7 @@ hsd.define('primitives', './primitives');
 hsd.define('Address', './primitives/address');
 hsd.define('Block', './primitives/block');
 hsd.define('Coin', './primitives/coin');
+hsd.define('Covenant', './primitives/covenant.js');
```

Comment:
You can leave off the `.js` suffix, it would better match the rest of the style

Reply:
Good catch